docs/logging/logging.md

```diff
@@ -0,0 +1,140 @@
+---
+sidebar_position: 1
+sidebar_label: Logging
+title: ""
+keywords:
+- Harvester
+- harvester
+- Logging
+---
+
+# Harvester Logging
+
+_Available as of v1.1.0_
+
+The Harvester logging infrastructure allows you to aggregate Harvester logs into an external service such as [Elasticsearch](https://www.elastic.co/elasticsearch/),
+[Grafana Loki](https://grafana.com/oss/loki/), or [Splunk](https://www.splunk.com/).
+
+## Collected Logs
+
+While users are able to configure and modify where the aggregated logs are sent, they cannot change which logs are
+aggregated. See below for a list of which logs are collected:
+ - Logs from all cluster `Pods`
+ - Kernel logs from each node
+ - Logs from select systemd services from each node
+   - `rke2-server`
+   - `rke2-agent`
+   - `rancherd`
+   - `rancher-system-agent`
+   - `wicked`
+   - `iscsid`
+
+## Configuring Log Destinations
+
+Logging is backed by the [Banzai Cloud Logging Operator](https://banzaicloud.com/docs/one-eye/logging-operator/), and so
+is controlled by [`Flows`/`ClusterFlows`](https://banzaicloud.com/docs/one-eye/logging-operator/configuration/flow/) and
+[`Outputs`/`ClusterOutputs`](https://banzaicloud.com/docs/one-eye/logging-operator/configuration/output/). You can route
+and filter logs as you like by applying these `CRD`s to the Harvester cluster.
+
+When applying new `Ouptuts` and `Flows` to the cluster, it can take some time for the logging operator to effectively
+apply them. So please allow a few minutes for the logs to start flowing.
```

Comment:
```suggestion
When applying new `Ouptuts` and `Flows` to the cluster, it can take some time for the logging operator to effectively apply them. So please allow a few minutes for the logs to start flowing.
```

Reply:
multiline